Dockerfile.dashing

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 The Cartographer Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM osrf/ros:dashing-desktop
+
+ARG CARTOGRAPHER_VERSION=master
```

Comment:
I'm not really familiar with how cartographer CI generally works, but shouldn't this be `ros2-dashing`?  Same for Dockerfile.foxy below.

Reply:
This just means that we build against `master` of the cartographer core library.